src/frontends/common/include/openvino/frontend/manager.hpp

```diff
@@ -75,6 +77,10 @@ class FRONTEND_API FrontEndManager final {
     FrontEnd::Ptr load_by_model_impl(const std::vector<ov::Any>& variants);
 
     std::unique_ptr<Impl> m_impl;
+
+    friend FRONTEND_API void ov::shutdown();
+    /// \brief Shutdown the manager by try releasing frontend libraries
+    static void shutdown();
```

Comment:
why do we need static method here?

Reply:
Because is called from free function `ov::openvino_shutdown()` and there is no front end object.